partcad-cli/src/partcad_cli/click/commands/import.py

```
@@ -0,0 +1,48 @@
+import shutil
```

Ruff (0.8.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)